content/40.units_and_quantities.md

```diff
@@ -1,269 +1,682 @@
+For these cases, it is recommended to convert atomic electromagnetic units separately first in
+equations, if necessary.
 
-We construct a `Unit` object by providing a string containing atomic unit symbols, combined with operations in Python syntax, and the registry those atomic unit symbols are defined in.
-We use sympy's string parsing features to create the unit expression from the user-provided string.
+### Physical Constants
 
-`Unit` objects are associated with four instance members, a unit `Expression` object, a dimensionality `Expression` object, a `UnitRegistry` instance, and a scalar conversion factor to CGS units.
-These data are available for a `Unit` object by accessing the `expr`, `dimensions`, `registry`, and `cgs_value` attributes, respectively.
+`unyt` contains a large number of built-in physical constants and 
+properties of astronomical objects as `unyt_quantity` instances, which
+can be used in computations. The simplest way to use them is to import
+them directly from the `unyt` namespace:
 
-`Unit` provides the methods `same_dimensions_as`, which returns True if passed a `Unit` object that has equivalent dimensions, `get_cgs_equivalent`, which returns the equivalent cgs base units of the `Unit`, and the `is_code_unit` property, which is `True` if the unit is composed purely of code units and `False` otherwise.
-`Unit` also defines the `mul`, `div`, `pow`, and `eq` operations with other unit objects, making it easy to compose compound units algebraically.
+```python
+>>> from unyt import clight, kboltz, h, Mearth, G
+>>> for const in [clight, kboltz, h, Mearth, G]:
+...     print(const)
+299792458.0 m/s
+1.3806488e-23 kg*m**2/(K*s**2)
+6.62606957e-34 kg*m**2/s
+6.045644495102106e+24 kg
+6.67408e-11 m**3/(kg*s**2)
+```
 
-The `UnitRegistry` class provides the `add`, `remove`, and `modify` methods which allows users to add, remove, and modify atomic unit definitions present in `UnitRegistry` objects.
-A dictionary lookup table is also attached to the `UnitRegistry` object, providing an interface to look up unit symbols.
-In general, unit registries should only be adjusted inside of a code frontend, since otherwise quantities and arrays might be created with inconsistent unit metadata.
-Once a unit object is created, it will not receive updates if the original unit registry is modified.
+Here's an example of using constants in a computation:
+```python
+>>> import numpy as np
+>>> from unyt import unyt_quantity
+>>> from unyt import Msun, G
+>>> d = unyt_quantity(1.0, "AU")
+>>> T = np.sqrt(4.0*np.pi**2*d**3/(G*Msun)) # Kepler's third law
+>>> print(T.to("yr"))
+1.000033863000043 yr
+```
 
-### Creating YTArray and YTQuantity instances {#sec:creating-ytarray-and-ytquantity-instances}
+Some physical constants are represented by Greek letters or other characters--these can be imported either using ASCII characters or
+their non-ASCII representations:
```

Comment:
It might be good to also demonstrate this part, as the non-ASCII names may not be obvious to the reader (they aren't to me)

Reply:
Done